js/src/dom/eventHandler.js

```diff
@@ -234,14 +232,31 @@ const EventHandler = {
 
     const typeEvent   = event.replace(stripNameRegex, '')
     const isNative    = nativeEvents.indexOf(typeEvent) > -1
+
+    const $ = Util.jQuery
+    let jQueryEvent
+
+    let bubbles = true
+    let nativeDispatch = true
+    let defaultPrevented = false
+
+    if (!isNative && typeof jQuery !== 'undefined') {
```

Comment:
here you check for `jQuery !== undefined` it should be better with `$` which come from your `Util.jQuery` getter

Reply:
You're right. Fixed.